resources/js/app.js

```diff
@@ -534,3 +534,17 @@ function Kanban() {
         }
     }
 }
+
+$(function(){
```

Comment:
This code can be removed and managed from the template code instead of JS code.

Also, if you want to add JS code, you don't need to add another `$(function(){});` use the one already exists in the JS file.